Library/Formula/ocproxy.rb

```diff
@@ -0,0 +1,27 @@
+class Ocproxy < Formula
+  desc "user-level SOCKS and port forwarding proxy"
+  homepage "https://github.com/cernekee/ocproxy"
+  url "https://github.com/cernekee/ocproxy/archive/v1.50.tar.gz"
+  sha256 "b061d59e0b5729d7a8950d8d4e0004a9739c092a4d5daf58a3bc61b177b24e4f"
+
+  head do
+    url "https://github.com/cernekee/ocproxy.git"
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libevent"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    `#{bin}/ocproxy 2>&1` =~ /VPNFD.is.not.set/
```

Comment:
Use `assert_match` here instead. Also, as it's a proxy I appreciate it's probably not but: can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!